src/components/TopBar.js

```diff
@@ -46,11 +49,13 @@ const mapDispatchToProps = dispatch => ({
     logout: () => dispatch(logout())
 });
 
-const TopBarWrapper = styled.section`
-    width: 100%;
-    background-color: ${Colors.black};
+const Inner = styled.section`
+    align-items: center;
```

Comment:
For CSS properties please use the following: Positioning > Display & Box Model > Color > Text > Other 
https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/